src/components/index.d.ts

```diff
@@ -15,6 +15,64 @@ export { Layout } from './Layout/index.js'
 // TODO -- DELETE THIS -- NOT GOING TO EXPORT DEPRECATED MEDIA
 // export { Media } from './Media'
 
+declare module 'react' {
+  interface HTMLAttributes<T> extends AriaAttributes, DOMAttributes<T> {
+    name?: string
+  }
+}
+
+interface HeaderProps {
+  name: string
+  leftChildren: React.ReactNode
+  rightChildren: React.ReactNode
+}
+
+/**
+ * Base `Header` component that provides fixed positioning and renders
+ * left and right side children. For ease of extension, `Header` is
+ * intentionally much simpler then the UniversalNavbar component.
+ * @see See [UniversalNavbar](https://github.com/getethos/ethos-design-system/blob/master/src/components/UniversalNavbar/UniversalNavbar.js)
+ *
+ * @public
+ *
+ * @param {object} props - Component Props
+ * @prop {string} props.name - Unique name of header.
+ * @prop {React.ReactNode} props.leftChildren - children to be rendered on left side of header
+ * @prop {React.ReactNode} props.rightChildren - children to be rendered on right side of header
+ *
+ * @return {JSX.Element}
+ */
+export declare const Header: {
+  ({ name, leftChildren, rightChildren }: HeaderProps): JSX.Element
+  propTypes: {
+    /** a required unique name for the header */
+    name: string
+    /** Optional children to render on left side of header */
+    leftChildren: React.ReactNode
+    /** Optional children to render on right side of header */
+    rightChildren: React.ReactNode
+  }
+  defaultProps: {
+    leftChildren: any
```

Comment:
It's defined as `undefined` in the implementation. I'm honestly afraid to touch this as it was modified from running `tsc --jsx react --declaration` which is usually my first step in creating these type definitions. Also, we won't be using `defaultProps` when we consume this so it kind of doesn't matter.

Someday EDS will be fully in Typescript (or not?) lol